packages/legacy-client/src/client/objectSets/mapPropertiesToMultipleAggregationProperties.ts

```diff
@@ -35,11 +35,22 @@ export function mapPropertiesToMultipleAggregationProperties<
         case "datetime":
           acc[property] = LocalDatePropertyMetric(property);
           break;
+        case "timestamp":
+          acc[property] = LocalDatePropertyMetric(property);
+          break;
 
+        case "decimal":
         case "double":
+        case "float":
         case "integer":
+        case "long":
+        case "short":
           acc[property] = NumericPropertyMetric(property);
 
+        case "byte":
+        case "attachment":
+        case "geopoint":
+        case "geoshape":
         case "boolean":
         case "string":
```

Comment:
I was kind of surprised to see byte and boolean down here instead of the block with the other numeric types.

Reply:
Byte is a good catch, though I'm not sure if it is aggregatable